technical_architecture/bundles/pim_bundles.rst.inc

```diff
@@ -8,7 +8,6 @@
 .. _FilterBundle: https://github.com/akeneo/pim-community-dev/tree/1.7/src/Pim/Bundle/FilterBundle
 .. _ImportExportBundle: https://github.com/akeneo/pim-community-dev/tree/1.7/src/Pim/Bundle/ImportExportBundle
 .. _InstallerBundle: https://github.com/akeneo/pim-community-dev/tree/1.7/src/Pim/Bundle/InstallerBundle
-.. _JsFormValidationBundle: https://github.com/akeneo/pim-community-dev/tree/1.7/src/Pim/Bundle/JsFormValidationBundle
```

Comment:
The other links should be updated to the right version of the PIM, but I will not do that in this PR. The goal is only to have a green CI here.